platform/jvm/AppleCodeSigner.kt

```diff
@@ -0,0 +1,245 @@
+package com.vaticle.bazel.distribution.platform.jvm
```

Comment:
This file contains the `AppleCodeSigner` class which provides all the functionality required to perform Apple code signing on the various pieces of a JVM platform image.

Apple code signing is required in order for an application to pass Apple's security checks when it is being downloaded and installed on a machine different to the one that built it.

We use it to sign our MacOS images.